package/yast2.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Fri Apr 24 14:53:03 UTC 2020 - Knut Anderssen <kanderssen@suse.com>
+
+- Avoid using systemctl calls when already started with the
+  installation and thus, running inside the chroot (bsc#1168849)
+- 4.2.83
```

Comment:
so we still in sync so we do not need minor bump, right?

Reply:
exactly